src/js/components/visualization/ChartComponent.jsx

```diff
@@ -83,15 +85,13 @@ export class ChartComponent extends Component {
         data={chartData}
         drilldown={drillDownFieldName != null}
         options={options}
-        ref='chart'
+        ref={(ref) => (this.chart = ref)}
```

Comment:
I saw this in the React documentation. I thought it was a little cleaner instead of having to reference the 'refs' object each time to get a reference. I don't mind either way though. We could also use underscores to ensure that there are not confliccts. We should probably figure out which way we prefer and stick with it to be consistent